src/Restaurant.js

```diff
@@ -1,14 +1,33 @@
-import React from "react";
+import React, { useState }from "react";
 
+const Order = (props) => {
+  const [orders, setOrders] = useState(0);
+
+  function orderOne() {
+    setOrders(orders + 1);
+  }
+  return (
+    <li>
+      {props.orderType}: {orders}
+      <RestaurantButton orderit={orderOne} />
+    </li>
+  );
+
+}
+
+const RestaurantButton = (props) => {
+  return (
+  <button className="btn btn-primary" onClick={props.orderit}>Add</button>
+  );
+}
 const Restaurant = () => {
-  const pizzas = 0;
   return (
     <div>
       <h3>Restaurant Orders</h3>
       <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+        <Order orderType="pizzas" />
+        <Order orderType="salads" />
+        <Order orderType="chocolate" />
       </ul>
     </div>
   );
```

Comment:
Generally its better to split your components into seperate files, it is easier to see whats going on 😄